scripts/mem_usage.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2014-2017, Linaro Limited
+# All rights reserved.
+
```

Comment:
I guess we could afford one line of spdx license description too.

Reply:
That was my intent, then I forgot to add it. Will do.